docs/book/README.md

```diff
@@ -28,7 +28,8 @@ This document will cover both the in-tree and out-of-tree vSphere integrations f
 * [Cloud Config Spec](cloud_config.md)
 
 ## Tutorials
+### vSphere 6.7U3 specific tutorials
+
+* [Deploying a K8s cluster (v1.14.2) with CSI/CPI on vSphere 6.7U3 with kubeadm](./tutorials/kubernetes-on-vsphere-1-14-2-with-kubeadm.md)
+* [Deploying CPI and CSI with Zones Topology](./tutorials/deploying_ccm_and_csi_with_multi_dc_vc_aka_zones.md) - needs to be revalidated - CJH
 
-* [Running a Kubernetes cluster on vSphere with kubeadm](./tutorials/kubernetes-on-vsphere-with-kubeadm.md)
-* [Deploying the out-of-tree vSphere Cloud Provider](./tutorials/deploying_cloud_provider_vsphere_with_rbac.md)
```

Comment:
I don't think it hurts to leave this here for now, at least to avoid dead links and if we want to reference just the CCM deployment steps. Thoughts?

Reply:
I agree. There are definitely some use cases where you might only want to deploy CPI and not CSI.

Reply:
OK - added back in.